src/Tribe/Validate.php

```diff
@@ -227,6 +227,20 @@ public function positive_int() {
 		}
 
 		/**
+		 * validates a field as being an integer
+		 *
+		 * @return stdClass validation result object
+		 */
+		public function int() {
```

Comment:
Exactly what @barryhughes mentioned, I mean. Thinking about the scope with 32bit systems and the fact that this uses a Regular Expression to check so It would validate any numeric value, not only valid integers.